README.md

```diff
@@ -1 +1,22 @@
-# golang-training-Library
\ No newline at end of file
+## Description
+
+Application for get books with such operations as in table below:
+
+
+|             Path            | Method | Description                           | Body example                                                                                                                                                                                                                     |
+|:---------------------------:|--------|---------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| /books                   | GET    | get all books                      | [{"BookId":1,"AuthorId":2,"PublisherId":1,"NameOfBook":"Belka","YearOfPublication":"2020-10-10", "BookVolume":20, "Number":1},{"BookId":2,"AuthorId":1,"PublisherId":4,"NameOfBook":"Strelka","YearOfPublication":"2021-12-21", "BookVolume":220, "Number":11},{"BookId":2,"AuthorId":3,"PublisherId":4,"NameOfBook":"Space","YearOfPublication":"2010-10-10", "BookVolume":202, "Number":11}] |
```

Comment:
could you show json body in the following way:
```json
Here goes your json object definition
```
I took this answer from here: https://stackoverflow.com/questions/14901245/how-to-style-a-json-block-in-github-wiki